lib/Doctrine/ORM/Cache/Region/FileLockRegion.php

```diff
@@ -59,7 +59,7 @@ class FileLockRegion implements ConcurrentRegion
     /** @var string */
     private $directory;
 
-    /** @var int */
+    /** @var string */
```

Comment:
https://github.com/doctrine/orm/blob/d6f926d8a17b572e0ab673a185036058b8e93afd/lib/Doctrine/ORM/Cache/Region/FileLockRegion.php#L111

Maybe you're looking for the type numeric-string?